public/Disable-DbaReplDistributor.ps1

```diff
@@ -0,0 +1,102 @@
+function Disable-DbaReplDistributor {
+    <#
+    .SYNOPSIS
+        Disables replication distribution for the target SQL instances.
```

Comment:
```suggestion
        Disables the Distributor feature for replication on the specified SQL Server instance(s).
```

Reply:
@jpomfret question on this one because documentation is stating for the method used that it actually uninstalls all replication publishing and distribution on the current instance. This is not "disabling", should we call this `Uninstall-DbaReplication` instead?

Ref: https://learn.microsoft.com/en-us/dotnet/api/microsoft.sqlserver.replication.replicationserver.uninstalldistributor